src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/InfoboxBuilderTests.java

```diff
@@ -54,31 +60,41 @@ public void addingComponents() {
 
   @Execute(asUser = User.USER)
   public void savingTemplate() {
-    new InfoboxBuilderPage().openExisting("InfoboxBuilderSavingTemplate")
-        .deleteRowUsingButton(0)
-        .addRowComponent()
-        .clickPublish()
-        .isSpinnerPresent();
+    Sidebar builderSidebar = new Sidebar();
+    InfoboxBuilderPage builderPage = new InfoboxBuilderPage();
+
+    builderPage.openExisting("InfoboxBuilderSavingTemplate").selectRowWithIndex(0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "InfoboxBuilderSavingTemplate" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)